<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoodsImportDialog</class>
 <widget class="QDialog" name="GoodsImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QTableView" name="invoiceView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>641</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveGoodsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>添加商品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calcButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>680</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>生成价格</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delGoodsButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>680</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>删除商品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>680</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>保存退出</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>641</width>
     <height>56</height>
    </rect>
   </property>
   <property name="title">
    <string>商品卷</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>24</y>
      <width>60</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>汇率</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cExchangeRateEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>24</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>125</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="couponDiscountEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>24</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>125</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>24</y>
      <width>60</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>折扣</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="couponEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>24</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>125</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>60</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>金额</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>641</width>
     <height>56</height>
    </rect>
   </property>
   <property name="title">
    <string>现金</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>60</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>金额</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="payPriceEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>24</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>125</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>24</y>
      <width>60</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>汇率</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exchangeRateEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>24</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>125</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>24</y>
      <width>60</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>找零</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="changeEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>24</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>125</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>641</width>
     <height>56</height>
    </rect>
   </property>
   <property name="title">
    <string>刷卡</string>
   </property>
   <widget class="QLineEdit" name="creditCardEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>24</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>125</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>31</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>金额</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>641</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>商品</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>64</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>代号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>金额</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="goodsIdEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="goodsPriceEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;名称&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="goodsNameEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>数量</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="goodsCountEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>31</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>属性</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="attributeEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>66</height>
    </rect>
   </property>
   <property name="title">
    <string>小票信息</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>60</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>日期</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>28</y>
      <width>125</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>125</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy/M/d</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>32</y>
      <width>60</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>商场名</string>
    </property>
   </widget>
   <widget class="QComboBox" name="shopNameBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>28</y>
      <width>125</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>125</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>680</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <zorder>groupBox_5</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>invoiceView</zorder>
  <zorder>saveGoodsButton</zorder>
  <zorder>calcButton</zorder>
  <zorder>delGoodsButton</zorder>
  <zorder>saveButton</zorder>
  <zorder>cancelButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
